To: vim_dev@googlegroups.com
Subject: Patch 7.3.22
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.222
Problem:    Warning for building GvimExt.
Solution:   Comment-out the DESCRIPTION line. (Mike Williams)
Files:      src/GvimExt/gvimext.def, src/GvimExt/gvimext_ming.def


*** ../mercurial/vim73/src/GvimExt/gvimext.def	2010-05-15 21:22:11.000000000 +0200
--- src/GvimExt/gvimext.def	2011-06-15 03:50:55.000000000 +0200
***************
*** 1,7 ****
  ;gvimdef.def : Declares the module parameters for the DLL.
  
  LIBRARY      gvimext
! DESCRIPTION  'Vim Shell Extension'
  
  EXPORTS
      DllCanUnloadNow private
--- 1,7 ----
  ;gvimdef.def : Declares the module parameters for the DLL.
  
  LIBRARY      gvimext
! ; DESCRIPTION  'Vim Shell Extension'
  
  EXPORTS
      DllCanUnloadNow private
*** ../mercurial/vim73/src/GvimExt/gvimext_ming.def	2010-05-15 21:22:11.000000000 +0200
--- src/GvimExt/gvimext_ming.def	2011-06-15 03:51:05.000000000 +0200
***************
*** 3,9 ****
  ;Hence this is the same file as gvimext.def with private removed
  
  LIBRARY      gvimext
! DESCRIPTION  'Vim Shell Extension build with MinGW'
  
  EXPORTS
      DllCanUnloadNow = DllCanUnloadNow@0
--- 3,9 ----
  ;Hence this is the same file as gvimext.def with private removed
  
  LIBRARY      gvimext
! ; DESCRIPTION  'Vim Shell Extension build with MinGW'
  
  EXPORTS
      DllCanUnloadNow = DllCanUnloadNow@0
*** ../vim-7.3.221/src/version.c	2011-06-19 01:14:23.000000000 +0200
--- src/version.c	2011-06-19 01:26:57.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     222,
  /**/

-- 
What is the difference between a professional and an amateur?
The ark was built by an amateur; professionals gave us the Titanic.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
